invenio/modules/oauthclient/handlers.py

```diff
@@ -209,7 +209,9 @@ def authorized_signup_handler(resp, remote, *args, **kwargs):
                 return redirect(url_for(
                     ".signup",
                     remote_app=remote.name,
-                    next=request.args.get('next', '/')
+                    next=request.args.get('next', '/'),
+                    external_id=account_info.get("external_id"),
```

Comment:
I think there's a security issue here. External id/method are passed in the request args to the form, and the code is then later relying on those values, meaning that the user can modified in between and thus fake them.